spec/models/cart_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+
+RSpec.describe "cart", type: :model do
+  it 'stores items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.